tests/plugins/example/__init__.py

```diff
@@ -0,0 +1,15 @@
+#
+# Copyright (C) 2020 Satoru SATOH
+#
+# SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Can you please stay consistent with other license headers, even in the plugin example code?

Reply:
Same as above.